data/xml/1978.tc.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="1978.tc">
+  <volume id="1" ingest-date="2021-08-04">
+    <meta>
+      <booktitle>Translating and the Computer</booktitle>
+      <publisher>Aslib Proceedings</publisher>
+      <address>London, UK</address>
+      <month>November 14</month>
+      <year>1978</year>
+      <editor><first>Barbara M.</first><last>Snell</last></editor>
+    </meta>
+    <frontmatter>
+      <url hash="390aee7d">1978.tc-1.0</url>
+      <bibkey>tc-1978-translating</bibkey>
+    </frontmatter>
+    <paper id="1">
+      <title>Multilingual communication: Chairman’s introductory review of translating and the computer</title>
+      <author><first>J.C.</first><last>Sager</last></author>
```

Comment:
This is a bit of a pain, but I think we should open the PDFs and expand the names.

Reply:
Okay, I'll do that for the future editions. Thanks for doing this edition!